deployer/aws-lambda/content-origin-request/index.js

```diff
@@ -225,11 +225,15 @@ export async function handler(event) {
     request.origin.custom &&
     request.origin.custom.domainName.includes("s3")
   ) {
-    // Rewrite the URI to match the keys in S3.
-    // NOTE: The incoming URI should remain URI-encoded. However, it
-    // must be passed to slugToFolder as decoded version to lowercase
-    // non-ascii symbols and sanitize symbols like ":".
-    request.uri = encodePath(slugToFolder(decodedUri));
+    // If this is bcd request we must not change the uri as they are
+    // case sensitive.
+    if (!decodedUri.startsWith("/bcd/")) {
+      // Rewrite the URI to match the keys in S3.
+      // NOTE: The incoming URI should remain URI-encoded. However, it
+      // must be passed to slugToFolder as decoded version to lowercase
+      // non-ascii symbols and sanitize symbols like ":".
+      request.uri = encodePath(slugToFolder(decodedUri));
+    }
```

Comment:
Nit: A newline separator would make this more readable.

```suggestion
    }

```